ISSUE_TEMPLATE.md

```diff
@@ -25,17 +22,15 @@ an interesting problem, rather than being redirected to Stack Overflow.
 - **Exact command to reproduce**:
 
 You can collect some of this information using our environment capture script:
+
 https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh
 
-You can collect the TensorFlow version with
-```sh
-python -c "import tensorflow as tf; print (tf.GIT_VERSION, tf.VERSION)"
-```
+You can obtain the TensorFlow version with
+
+python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"
```

Comment:
The reasoning for this is that when users see this template, they will see the unformatted markdown and not the formatted version. I wanted to make it easier for them to copy-paste this command into their terminals without the shell tags.